internal/provider/types/resource/services.go

```diff
@@ -161,17 +161,18 @@ var StartCommand = schema.StringAttribute{
 }
 
 var MaintenanceMode = schema.SingleNestedAttribute{
-	Optional:    true,
```

Comment:
Won't this prevent users from defining maintenance mode?

Reply:
Yes. Good catch! Should be fixed now